setup.py

```diff
@@ -6,9 +6,9 @@
 
 
 setup(
-    name="jupyter-desktop-server",
+    name="jupyter-remote-desktop-proxy",
     packages=find_packages(),
-    version='0.1.3',
+    version='0.2.0',
```

Comment:
```suggestion
    version='1.0.0',
```

I'm strongly opinionated towards going for 1.0.0 as soon as possible, such as now. Can we?

It improves the ease of maintenance when we can just follow semver2 meaning of version changes, and it is positive for users as well if they can better guess the impact of a change when version goes from something 1.0.0 to something else.

This is what semver.org suggests, but no matter what for me as a maintainer and user, going 1.0.0 is a clear win win.

![image](https://user-images.githubusercontent.com/3837114/212498363-f5ee5a1b-1c13-4586-9c6a-fb110d8715de.png)

Reply:
Sure thing!